src/libslic3r/PrintConfig.cpp

```diff
@@ -1784,7 +1784,7 @@ void PrintConfigDef::init_fff_params()
     def->set_default_value(new ConfigOptionBool(false));
 
     def = this->add("fan_speedup_time", coFloat);
-    def->label = L("Fan Speedup Time");
+    def->label = L("");
```

Comment:
Thanks, @scottmudge, can you remove the `L` so that gettext won't complain?
Or simply comment out this line and leave comment specify the reason?